pkg/sdk/go/groups.go

```diff
@@ -9,6 +9,7 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/absmach/magistrala/internal/apiutil"
```

Comment:
Add id check for `getGroups`, `GroupPermissions`, `changeGroupStatus`, `AddUserToGroup`, `RemoveUserFromGroup`, `ListGroupUsers`, `ListGroupUsers`, ``

Reply:
The id check for these is done at api level.